src/patternfly/components/List/list.scss

```diff
@@ -1,54 +1,40 @@
 @import "../../patternfly-utilities";
 
 .pf-c-list {
-  // Inline modifier variables
-  --pf-c-list--m-inline--MarginRight: var(--pf-global--spacer--md);
-
-  // Grid modifier variables
-  --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
-
-  // Content variables (no modifier)
-  --pf-c-content--li--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
-  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
-  --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
-
-  &.pf-m-inline {
-    display: flex;
-    flex-wrap: wrap;
+  --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-list--ul--nested--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-list--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
+  --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
+
+  padding-left: var(--pf-c-list--PaddingLeft);
+
+  ol,
+  ul {
+    margin: var(--pf-c-list--ul--nested--MarginTop) 0 0 var(--pf-c-list--ul--nested--MarginLeft);
+  }
 
-    & li:not(:last-child) {
-      margin-right: var(--pf-c-list--m-inline--MarginRight);
-    }
+  li + li {
+    margin-top: var(--pf-c-list--li--MarginTop);
   }
 
-  &.pf-m-grid {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
-    grid-gap: var(--pf-c-list--m-grid--GridGap);
+  &:not(ol) {
```

Comment:
could this just be `ul`?

Reply:
oh nevermind. After looking at the rest of the file I think you can change this to `@at-root ul#{&}:not(.pf-m-inline)` and then remove `--pf-c-list--ul--ListStyle: none` in the inline list